Automattic/jetpack
# P2: rename the `ready.o2` event to `ready_p2`

Differential Revision: D54524-code

This commit syncs r219004-wpcom.

#### Changes proposed in this Pull Request:

jQuery 3.5.1 handles the `ready.o2` event in terms of the `ready` event, thus breaks the P2

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

1. update your sandbox to jQuery 3.5.1 ( D54601 )
1. apply D54878 before this patch
1. sandbox a P2
1. check whether there are any errors


#### Proposed changelog entry for your changes:

* N/A
